starlite/middleware/base.py

```diff
@@ -49,7 +53,7 @@ def __init__(self, middleware: Callable[..., "ASGIApp"], *args: Any, **kwargs: A
             *args: Positional arguments to pass to the callable.
             **kwargs: Key word arguments to pass to the callable.
 
-        Notes:
+        !!! Note
```

Comment:
This change is to make the admonition default open - see: https://github.com/mkdocstrings/python/issues/22

Reply:
Do you mean when you type it you get a tab completion hint for `Notes:` but not one for `!!! Note`?

Reply:
This is the way it is with standard google docstrings:

<img width="940" alt="Screenshot 2022-08-31 at 13 54 44" src="https://user-images.githubusercontent.com/30733348/187672770-5646ce1b-b85e-4937-9fea-828f78b3f42c.png">

And this is how this looks with the admonition

<img width="951" alt="Screenshot 2022-08-31 at 13 54 59" src="https://user-images.githubusercontent.com/30733348/187672835-4125a6fb-d229-472f-84ef-5649f19ac0b3.png">

Reply:
I see, well I suppose it depends what you want to optimize for - neither is perfect.

Reply:
and if instead we do this:

```markdown
        Notes:
            - The callable will be passed a kwarg `app`, which is the next ASGI app to call in the middleware stack.
            It therefore must define such a kwarg.
```

It looks like this:

<img width="939" alt="Screenshot 2022-08-31 at 13 56 45" src="https://user-images.githubusercontent.com/30733348/187673075-b2dc63c9-bae5-48e6-b5d6-7fb232415276.png">

Which I think is the best one

Reply:
This is from live docs:

![image](https://user-images.githubusercontent.com/20659309/187676757-bd2c4c3e-2e98-4a33-a757-af8780b6ce12.png)

This is my local:

![image](https://user-images.githubusercontent.com/20659309/187677102-bbdaa497-ce63-4028-9745-90a3f0d0f431.png)

Reply:
We can just leave it and try and get it sorted upstream?

Reply:
ok

Reply:
```suggestion
        Notes:
```